docs/_data/previous_versions.yml

```diff
@@ -1,5 +1,5 @@
 - title: 0.0.15
-  url: /docs
+  url: /docs/0_0_15
```

Comment:
Does this need to be updated to actually include 0.0.16 as well. I can't quite remember how it works...

Reply:
yes